_inc/client/plans/single-product-backup.jsx

```diff
@@ -179,52 +288,20 @@ class SingleProductBackupBody extends React.Component {
 		} );
 	};
 
-	handleLandingPageLinkClick = selectedBackupType => () => {
-		analytics.tracks.recordJetpackClick( {
-			target: 'landing-page-link',
-			feature: 'single-product-backup',
-			extra: selectedBackupType,
-		} );
-	};
-
 	render() {
-		const {
-			backupOptions,
-			billingTimeFrame,
-			currencyCode,
-			selectedBackupType,
-			upgradeLinks,
-		} = this.props;
+		const { backupOptions, billingTimeFrame, currencyCode, selectedBackupType } = this.props;
 
-		// TODO: Move out those somewhere else to make this a flexible and fully reusable component.
-		const upgradeTitles = {
-			'real-time': __( 'Upgrade to Real-Time Backups' ),
-			daily: __( 'Upgrade to Daily Backups' ),
-		};
+		const selectedBackup = get( backupOptions, selectedBackupType, null );
 
 		return (
 			<React.Fragment>
-				<p>
-					{ __(
-						'Always-on backups ensure you never lose your site. Choose from real-time or daily backups. {{a}}Which one do I need?{{/a}}',
-						{
-							components: {
-								a: (
-									<ExternalLink
-										target="_blank"
-										href="https://jetpack.com/upgrade/backup/"
-										icon
-										iconSize={ 12 }
-										onClick={ this.handleLandingPageLinkClick( selectedBackupType ) }
-									/>
-								),
-							},
-						}
-					) }
-				</p>
-				<h4 className="single-product-backup__options-header">{ __( 'Backup options:' ) }</h4>
+				<p>{ __( 'Always-on backups ensure you never lose your site.' ) }</p>
```

Comment:
I think we could use some padding below this one.

Reply:
Addressed in f879054c0a1452457d34fb71bc02d744114fe4e6, but it doesn't look great to me?

Reply:
I think you should add some more 😉 

Right now we have:

![](https://cldup.com/0qW0IhQ7bq.png)

And I think that in the design, the space above and below this paragraph is equal.

Reply:
Okay, so I tweaked things in 131362ec28a949bbba7afbef9e15c6102fcd87cd:

![Screenshot on 2019-12-16 at 14_53_54](https://user-images.githubusercontent.com/1760168/70938482-3d69cc80-2014-11ea-86cf-b6304f83eece.png)

@tyxla - look good to you?

Reply:
@ChaosExAnima Thank you for refining the appearance of the card here.

I've got just one minor request: can you align the promo nudge copy to the left (for both mobile and desktop breakpoints) so that it matches the designs?

Reply:
Left-aligned the copy in https://github.com/Automattic/jetpack/pull/14203/commits/af110037d68ed8cee1da576fc3c8a07876e22dc2.

Regarding the vertical space - for wp-admin it seems that design doesn't suggest additional vertical space - should we keep it as-is?

Reply:
To me it seems that there is much more space before and after the promo nudge in the designs, but I'm happy to leave it as is. I'm sure this will go through a testing/design review at some point so we can address minor issues like this at the time.

Reply:
Thanks, I've added some more space below the promo nudge!